constants/misc_constants.asm

```diff
@@ -16,3 +16,6 @@ DEF TEXT_DELAY_SLOW   EQU %101 ; 5
 	const_def 6
 	const BIT_BATTLE_SHIFT     ; 6
 	const BIT_BATTLE_ANIMATION ; 7
+
+; D732 flags
```

Comment:
Don't add *more* address comments.

Reply:
As I explained on Discord, I had decided to not rename `wd732` and `wFlags_D733` because there hadn't been sufficient discussion on the appropriate replacements.

Reply:
In that case `; wd732 flags` would be the appropriate comment: using the label, which happens to itself still be unidentified.

Reply:
I was being consistent with the comment that accompanies the definition of `BIT_TEST_BATTLE` so I'll update both accordingly.

Reply:
This is still `; D732 flags`, it should use the label `wd732`.